gauche/kernels/string_kernels/sskkernel.py

```diff
@@ -0,0 +1,230 @@
+# GPU-compatible gpytorch version of the substring kernel implemented in https://github.com/henrymoss/BOSS/tree/master/boss/code/kernels/string
```

Comment:
Should we format the module-level docstring as in: 

https://github.com/leojklarner/gauche/blob/main/gauche/kernels/gnn_kernels/pretrained_kernel.py

for consistency across the codebase?

Reply:
updated both the top-level and function-level docstring with the new style